### PR TITLE
scan: added loading state to validator index page

### LINKED ISSUE #1564: Implement Loading state on Validator Index Page
Let's make a loading state as design.

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1564, #1969

### CODE REVIEW

File: scan/src/components/validator/ProposedBlocksTable.re [hunk 1]
```diff
@@ -30,107 +147,35 @@ let make = (~consensusAddress) =>
...
-             <VSpacing size={`px(40)} />
-             <Text block=true value="NO BLOCK" weight=Text.Regular color=Colors.blue4 />
-             <VSpacing size={`px(15)} />
-           </div>}
+      {switch (blocksSub) {
+       | Data(blocks) =>
+         blocks->Belt.Array.size > 0
+           ? <>
+               <TableHeader />
+               <BlocksTable blocks />
```
Comment (evilpeach) [inline scan/src/components/validator/ProposedBlocksTable.re]: Should `<TableHeader>` place outside of blocksSub ?

File: scan/src/components/validator/ReportsTable.re [hunk 2]
```diff
@@ -28,6 +28,225 @@ module Styles = {
...
+            <Col size=1.>
+              {reportDetails
+               ->Belt_Array.map(reportDetail => {
+                   <>
+                     <Row> <TypeID.DataSource id={reportDetail.dataSourceID} /> </Row>
+                     <VSpacing size=Spacing.sm />
+                     <VSpacing size=Spacing.xs />
+                   </>
+                 })
+               ->React.array}
```
Comment (evilpeach) [inline scan/src/components/validator/ReportsTable.re]: add key

File: scan/src/components/validator/ReportsTable.re [hunk 3]
```diff
@@ -28,6 +28,225 @@ module Styles = {
...
+            </Col>
+            <Col size=1.5>
+              {reportDetails
+               ->Belt_Array.map(({externalID}) => externalID)
+               ->Belt_Array.map(externalDataID => {
```
Comment (evilpeach) [inline scan/src/components/validator/ReportsTable.re:167]: add key

File: scan/src/components/validator/ReportsTable.re [hunk 4]
```diff
@@ -28,6 +28,225 @@ module Styles = {
...
+                     </div>
+                     <VSpacing size=Spacing.md />
+                   </>
+                 })
+               ->React.array}
+            </Col>
+            <Col size=2.2>
+              {reportDetails
+               ->Belt_Array.map(({data}) => data)
+               ->Belt_Array.map(value => {
```
Comment (evilpeach) [inline scan/src/components/validator/ReportsTable.re:183]: add key

File: scan/src/components/validator/DelegatorsTable.re [hunk 5]
```diff
@@ -41,103 +122,66 @@ let make = (~address) =>
...
-             <VSpacing size={`px(15)} />
-           </div>}
+      {switch (delegatorsSub) {
+       | Data(delegators) =>
+         delegators->Belt_Array.length > 0
+           ? <>
+               <Header />
+               {delegators
+                ->Belt.Array.map(({amount, sharePercentage, delegatorAddress}) => {
+                    <TBody>
```
Comment (evilpeach) [inline scan/src/components/validator/DelegatorsTable.re]: add key?

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Comment by evilpeach** [comment]
@tansawit Please fix conflict.

**Review by evilpeach — CHANGES_REQUESTED** [review]
Please fix 
1) image

2) image

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**